tests/components/ResultsCount.test.tsx

```diff
@@ -114,7 +122,22 @@ describe('Results count for universal search', () => {
 
   it('Renders nothing if there is no result', () => {
     mockAnswersState(mockedStateUniversalNoResult);
+    const { container } = render(<ResultsCount />);
+    expect(container.childNodes[0]).toBeEmptyDOMElement();
+  });
+});
+
+describe('Results count and pagination range for vertical search', () => {
+  it('Renders pagination range if pagination is required', () => {
+    mockAnswersState(mockedStateVerticalPaginationResult);
+    render(<ResultsCount />);
+    expect(screen.getByText('1 - 20 of 30 Results')).toBeDefined();
+  });
+
+  it('Does not render pagination range if there is no pagination', () => {
+    mockAnswersState(mockedStateVerticalMultiple);
     render(<ResultsCount />);
-    expect(screen.queryByText('0 Results')).toBeNull();
+    expect(screen.queryByText('1-2 of 2 Results')).toBeNull();
```

Comment:
this text needs to be updated right?

Reply:
this is the danger with checking that something is null

Reply:
can we check that the page has no text instead?

Reply:
yeh ty

Reply:
In this case, we are looking that the page has "2 Results" and not "1 - 2 of 2 Results" so the page would not be empty

Reply:
Instead, I found that there was an element that had text "2 Results" and I checked that the text content of that element was exactly "2 Results" and nothing else.